packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -143,4 +143,76 @@ class CupertinoIcons {
 
   /// Two right-facing intertwined arrows.
   static const IconData shuffle_thick = IconData(0xf221, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// Symbolizes a photo camera
+  static const IconData photo_camera = IconData(0xf3f5, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// Solid [photo_camera]
+  static const IconData photo_camera_solid = IconData(0xf3f6, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// Symbolizes a video camera
+  static const IconData video_camera = IconData(0xf4cc, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// Solid [video_camera]
+  static const IconData video_camera_solid = IconData(0xf4cd, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// Camera filled with two circular arrows, which indicate switching
+  static const IconData switch_camera = IconData(0xf49e, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// Solid [switch_camera]
+  static const IconData switch_camera_solid = IconData(0xf49f, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// Envelopes stacked facing forwards
+  static const IconData collections = IconData(0xf3c9, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// Solid [collections]
+  static const IconData collections_solid = IconData(0xf3ca, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// Single envelope, i.e. a folder
```

Comment:
Envelope for mail? Folder for files? I've no idea, from this description, what the icon is for. I'm actually less sure after seeing this documentation than I was when I just saw the icon name "folder". :-)